firmware/controllers/trigger/rpm_calculator.cpp

```diff
@@ -372,3 +372,10 @@ RpmCalculator::RpmCalculator() {
 
 #endif /* EFI_SHAFT_POSITION_INPUT */
 
+
+/*
+ * Local variables:
```

Comment:
@DonaldBecker @mck1117 what are these formatting rules used by? do we want in official repo?

Reply:
I crafted a `clang-format` file that holds pretty close to the existing formatting rules, if we want to use it.  Not sure where this inline config came from (clang-format or otherwise)

Reply:
The lines are formatting rules for emacs and a few other tools.

Reply:
should they be here, or is it possible to drop a formatting config file in the root or something?